modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -99,9 +100,9 @@ Returning to the Good Weather use case, imagine that the company uses the follow
 
 image::multiple-IdP-user-flow.png[User Flow for Configuring Multiple External Client Providers]
 
-As shown in the diagram, you first configure your external client provider and assign it to the appropriate business groups and environments. When you create or modify an API within a specific business group or environment, the new client provider is displayed. You can reselect the appropriate client provider. 
+As shown in the diagram, you first configure your external client provider and assign it to the appropriate environments. When you create or modify an API within a specific environment, the new client provider is displayed. You can reselect the appropriate client provider. 
 
-When you promote an API to another environment, the target environment must have the client provide configured as in the source environment. Otherwise, the promotion does not complete successfully. 
+When you promote an API to another environment, the target environment must have the client provider configured as in the source environment. Otherwise, a promoted API's IdP policy might not match the API's associated IdP. To resolve this mismatch, delete the IDP policy. 
```

Comment:
Promote?

Reply:
```suggestion
To promote an API from one environment to another, either both environments must use the same client provider or you must delete [which?] IdP policy to prevent a policy mismatch.
```

Reply:
Just an example trying to clarify the `[which?]` part of your suggestion:

When promoting an api instance with, let say, a pingFederate IDP and the `pingFederate access token enforcement` policy (IDP policies match with the IDP type) to an environment with openAM IDPs, the mentioned policy has to be removed for the promoted api to work because the promoted api will be associated with an openAM IDP.  Later, the user may want to apply the policy `openAM access token enforcement` to protect that new api.

If the target environment were associated with any pingFederate IDP and if the customer chose any of those IDPs to be associated with the promoted API, there wouldn't be any conflict with any applied policy because there wasn't any IDP type change.

Reply:
Fixed! Thanks.